src/main/java/org/jenkinsci/plugins/workflow/actions/ExecutorTaskInfoAction.java

```diff
@@ -0,0 +1,88 @@
+package org.jenkinsci.plugins.workflow.actions;
+
+import hudson.model.InvisibleAction;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Records information for a {@code node} block.
+ */
+public class ExecutorTaskInfoAction extends InvisibleAction implements PersistentAction {
```

Comment:
Ahh, okay, that makes sense then -- but that suggests we should be removing this Action once it has a proper Computer/Workspace to play with, then replacing it with the appropriate WorkspaceAction/etc that has missing info filled in.  Neh?

The behavior of the Action here just feels *wrong* because of this -- but I think this key runtime information should be readily exposed by API somehow (until we get an executor anyway).